pkg/sfu/forwarder.go

```diff
@@ -764,7 +864,7 @@ func (f *Forwarder) getTranslationParamsVideo(extPkt *buffer.ExtPacket, layer in
 		return tp, err
 	}
 
-	if f.vp8Munger == nil {
+	if f.vp8Munger == nil || len(extPkt.Packet.Payload) == 0 {
```

Comment:
Do not parse as VP8 padding only packets and forward them if it gets here. If padding only packets get here, it is following a gap in sequence number. As we do not know what temporal layer would have been in the gap, just forward unconditionally. We may be able to drop padding only packets even after a gap. Need some more testing/thinking. But, forwarding for now.